CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## v1.13.0 (27/09/2021)
+
+**Common**
+
+* Added a new `olp::client::EqualJitterBackdownStrategy` backdown strategy.
+* Changed protected cache behavior. Now it opens the cache in ReadWrite mode if possible and user passed default open options.
+* Now `olp::cache::DefaultCache::StorageOpenResult::ProtectedCacheCorrupted` also returned if there is more than 4 L0 leveldb files in protected cache.
+* Added a bool operator to ApiResponse class, a shortcut to `ApiResponse::IsSuccessful()`.
+* Removed arbitrary 2s delay on shutdown.
+* Now request isn't sent if bearer token is empty. Previously, if authentication failed for some reason, data request was still sent with empty token.
+* Fixed WinHttp request was not freed correctly after an error.
+* Fixed unicode compilation on windows.
+* Fixed `olp::Dir::FileExist()` method for windows.
```

Comment:
```suggestion
* Fixed `olp::Dir::FileExist()` method when compiled for Windows. Instead of checking the `FILE_ATTRIBUTE_NORMAL` attribute it now checks that the attributes are unequal to `INVALID_FILE_ATTRIBUTES` to assume a file is present on the filesystem.
```